packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer.tsx

```diff
@@ -77,10 +77,12 @@ const ContractDrawer = observer(
                 exit_spot={exit_tick_display_value}
                 is_accumulator={is_accumulator}
                 is_dark_theme={is_dark_theme}
+                is_history_tab_active={is_history_tab_active}
                 is_multiplier={is_multiplier}
                 is_open
                 is_turbos={is_turbos}
                 is_vanilla={is_vanilla}
+                current_lang_to_request_history={is_language_changing ? '' : current_language}
```

Comment:
```suggestion
                current_language={is_language_changing ? '' : current_language}
```
To make the prop name simple :D